2021-01-27 21:51:18,737 - CasaBlanca - ERROR: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
Traceback (most recent call last):
  File "C:\Users\pcc\Desktop\Escritorio\CasaBlanca\app.py", line 216, in nuevoUsuario
    nombre=request.form['nombre']
  File "c:\Users\pcc\Desktop\Escritorio\CasaBlanca\env\lib\site-packages\werkzeug\datastructures.py", line 442, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.BadRequestKeyError: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
2021-01-27 22:02:30,653 - CasaBlanca - ERROR: Not all parameters were used in the SQL statement
Traceback (most recent call last):
  File "C:\Users\pcc\Desktop\Escritorio\CasaBlanca\app.py", line 228, in nuevoUsuario
    ''',(nombre, apellido, cedula, email,hashedPass, rol))
  File "c:\Users\pcc\Desktop\Escritorio\CasaBlanca\env\lib\site-packages\mysql\connector\cursor.py", line 569, in execute
    "Not all parameters were used in the SQL statement")
mysql.connector.errors.ProgrammingError: Not all parameters were used in the SQL statement
2021-01-27 22:07:55,730 - CasaBlanca - ERROR: Not all parameters were used in the SQL statement
Traceback (most recent call last):
  File "C:\Users\pcc\Desktop\Escritorio\CasaBlanca\app.py", line 228, in nuevoUsuario
    ''',(cedula, nombre, apellido, email,hashedPass, rol))
  File "c:\Users\pcc\Desktop\Escritorio\CasaBlanca\env\lib\site-packages\mysql\connector\cursor.py", line 569, in execute
    "Not all parameters were used in the SQL statement")
mysql.connector.errors.ProgrammingError: Not all parameters were used in the SQL statement
2021-01-27 22:09:14,565 - CasaBlanca - ERROR: Not all parameters were used in the SQL statement
Traceback (most recent call last):
  File "C:\Users\pcc\Desktop\Escritorio\CasaBlanca\app.py", line 228, in nuevoUsuario
    ''',(cedula, nombre, apellido, email,hashedPass, rol))
  File "c:\Users\pcc\Desktop\Escritorio\CasaBlanca\env\lib\site-packages\mysql\connector\cursor.py", line 569, in execute
    "Not all parameters were used in the SQL statement")
mysql.connector.errors.ProgrammingError: Not all parameters were used in the SQL statement
